salt/netapi/__init__.py

```diff
@@ -157,6 +157,11 @@ def run(self, low):
                 "Invalid client specified: '{}'".format(low.get("client"))
             )
 
+        if low.get("client") not in self.opts.get("netapi_enable_clients"):
+            raise salt.exceptions.SaltInvocationError(
+                "Client disabled: '{}'".format(low.get("client"))
```

Comment:
Please improve this message:

- Inform the user they need to add the client to `netapi_enable_clients` in their config if they want to use this salt-api with the client they have specified.

Reply:
Done